src/v0/destinations/klaviyo/util.js

```diff
@@ -1,116 +1,75 @@
-const { httpGET } = require('../../../adapters/network');
-
+const { defaultRequestConfig } = require('rudder-transformer-cdk/build/utils');
 const { WhiteListedTraits } = require('../../../constants');
 
 const {
-  defaultRequestConfig,
   constructPayload,
   getFieldValueFromMessage,
-  isDefinedAndNotNull,
-  removeUndefinedValues,
   defaultPostRequestConfig,
   extractCustomFields,
   removeUndefinedAndNullValues,
+  defaultBatchRequestConfig,
 } = require('../../util');
 
-const { BASE_ENDPOINT, LIST_CONF, MAPPING_CONFIG, CONFIG_CATEGORIES } = require('./config');
-const {
-  getDynamicErrorType,
-  processAxiosResponse,
-} = require('../../../adapters/utils/networkUtils');
-const { NetworkError } = require('../../util/errorTypes');
-const tags = require('../../util/tags');
+const { BASE_ENDPOINT, MAPPING_CONFIG, CONFIG_CATEGORIES } = require('./config');
 const { JSON_MIME_TYPE } = require('../../util/constant');
 
 /**
- * This function is used to check if the user/profile already exists or not, if already exists unique person_id
- * for that user is getting returned else false is returned.
- * Docs: https://developers.klaviyo.com/en/reference/get-profile-id
- * @param {*} message
- * @param {*} Config
- * @returns
+ * This function is used for creating response for subscribing users to a particular list.
+ * DOCS: https://www.klaviyo.com/docs/api/v2/lists
  */
-const isProfileExist = async (message, { Config }) => {
-  const { privateApiKey } = Config;
-  const userIdentifiers = {
+const subscribeUserToList = (message, traitsInfo, destination) => {
```

Comment:
Can we refactor this method to solve the Code complexity in Sonar ?

Reply:
Updated